src/main/java/it/pagopa/pn/deliverypush/utils/AuthUtils.java

```diff
@@ -0,0 +1,116 @@
+package it.pagopa.pn.deliverypush.utils;
+
+import it.pagopa.pn.deliverypush.dto.ext.delivery.notification.NotificationInt;
+import it.pagopa.pn.deliverypush.dto.ext.mandate.MandateDtoInt;
+import it.pagopa.pn.deliverypush.exceptions.PnNotFoundException;
+import it.pagopa.pn.deliverypush.service.MandateService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+import java.time.Instant;
+import java.util.List;
+
+@Slf4j
+@Component
+public class AuthUtils {
+    private final MandateService mandateService;
+
+    public AuthUtils(MandateService mandateService) {
+        this.mandateService = mandateService;
+    }
+
+    public void checkAuthorization(NotificationInt notification, String senderRecipientId, String mandateId) {
```

Comment:
Questa classe e questo metodo hanno dei nomi troppo generici AuthUtils.checkAuthorization e non si capisce qual'è lo scopo. Ad esempio questa classe controlla se una notifica può essere vista da un destinatario o un delegato: _NotificationAccessAuth_ 
La stessa cosa è anche in pn-delivery? perché non mettere il codice in comune?